logging/jboss/src/main/resources/application.properties

```diff
@@ -8,3 +8,8 @@
 # By default min-level is set to DEBUG
 #quarkus.log.min-level=DEBUG
 quarkus.log.level=TRACE
+
+quarkus.log.file.enable=true
+quarkus.log.file.rotation.file-suffix=.yyyy-MM
+quarkus.log.file.rotation.max-file-size=100
```

Comment:
Can you add some comment that this size is for test populate both backup logs.

As someone can try to set bigger value or when adding some tests, the `checkLogRotationContent` can fail

I play with the value and hit the failing `checkLogRotationContent`

Reply:
updated